bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/dsf/Launch.java

```diff
@@ -217,6 +219,9 @@ public Process call() throws CoreException
 			// First set attribute to specify we want to create the gdb process.
 			// Bug 210366
 			Map<String, String> attributes = new HashMap<String, String>();
+			ILaunchConfigurationWorkingCopy workingCopy = this.getLaunchConfiguration().getWorkingCopy(); 
+			workingCopy.setAttribute(DebugPlugin.ATTR_PROCESS_FACTORY_ID, CustomIdfProcessFactory.ID);
+			workingCopy.doSave();
```

Comment:
will it work, if move these to provideDefaults() ?

Reply:
Yes fixed